### PR TITLE
Some minor fixes

### LINKED ISSUE #2570: Loading indicator
For some layers like https://map.geo.admin.ch/mobile.html?X=174472.32&Y=614224.87&zoom=4&topic=ech&lang=fr&bgLayer=ch.swisstopo.pixelkarte-farbe&layers_opacity=0.75&mobile=true&layers=ch.bafu.gewaesserschutzbereiche the tooltip query can take quite a long time. On mobile, we have a loading indicator. Could we have something equivalent on desktop ? Currently, the user clicks but doesn't know that something happens.
@davidoesch @gjn @oterral thoughts ?

### LINKED ISSUE #2811: add msgid for kml drawing
When current drawing is shared it appears with a title "drawing" in all lang
AFAIK, this is the KML <Document><name> tag - we discussed this as well during the dev  of drawing... with @oterral 

proposal
A) detect lang setting at the beginning of the draw session and set <Document><name> to

DE Zeichnung
FR Dessin
IT  Disegno
EN Drawing
RM Dissegn

B) otherwise set it to all lang to a unilang abbrevation like
"KML" or "public.geo.admin.ch"

my vote goes out for A)

### LINKED ISSUE #3014: 3D and tiny iframes
No idea how deal with this one here ---
http://www.sac-cas.ch/unterwegs/tourenplanung/naturvertraeglich-unterwegs.html
image

Expected result:

No overlapping 
Icons visible

### LINKED ISSUE #3271: Iframe does not accept swipe
Reproduce

Take 
https://map.geo.admin.ch/?topic=ech&lang=de&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.geologie-eiszeit-lgm-raster,ch.swisstopo.geologie-eiszeit-lgm,KML%7C%7Chttps:%2F%2Fpublic.geo.admin.ch%2FwwdHdWn-RVuqsMjwKS1HVw&X=191370.00&Y=759380.00&zoom=7&layers_visibility=true,false,false&swipe_ratio=0.59

go to embed

In the embed code / result
Swipe line does appear, but the LGM layer is not swiped
unbenannt

### LINKED ISSUE #3285: Add max kml size in ' file too big' popup message

### LINKED ISSUE #3291: [search field] Typo in German standard text in search field
UC: open app in German, look at the search field
Result: there is a slight typo in the standard text of the search field: **z.b. **Bundesplatz

Expected: **z.B.**
typo_search_field_de
Changed that already in the translation table.

### PR DESCRIPTION
**Pull request by oterral**
Fix #2570 
Fix #2811
Fix #3291 
Fix #3271 
Fix #3285 
Fix #3014 

[Test](https://mf-geoadmin3.dev.bgdi.ch/teo_fixes/?lang=de&topic=aviation&bgLayer=ch.swisstopo.pixelkarte-grau&X=189500.00&Y=660000.00&dev3d=true&zoom=1&debug&layers=ch.bazl.luftfahrthindernis,KML%7C%7Chttps:%2F%2Fpublic.dev.bgdi.ch%2F4cEd0VkIRr6AmmUEbHA6Og&catalogNodes=1379,1381,1514)

### CODE REVIEW

**Comment by gjn** [comment]
Code LGTM. @davidoesch Please test and give go.

**Comment by oterral** [comment]
jenkins test

**Comment by oterral** [comment]
@davidoesch  #2811 fixed

**Comment by oterral** [comment]
jenkins becames crazy?

File: src/components/feedback/FeedbackDirective.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ goog.require('ga_permalink');
             link: function(scope, element, attrs) {
               function validateSize(fileSize) {
                 if (fileSize > 10000000) { // 10 Mo
-                  alert($translate.instant('file_too_large') + ' (Max 10Mb)');
+                  alert($translate.instant('file_too_large') + ' (max. 10 Mo)');
```
Comment (gjn) [inline src/components/feedback/FeedbackDirective.js]: Please use english term when not translated. Mb is much more rocognized than Mo.

File: src/components/map/KmlService.js [hunk 2]
```diff
@@ -359,8 +359,8 @@ goog.require('ga_urlutils_service');
 
         // Test the validity of the file size
         this.isValidFileSize = function(fileSize) {
-          if (fileSize > 20000000) { // 20mo
-            alert($translate.instant('file_too_large'));
+          if (fileSize > 20000000) { // 20 Mo
+            alert($translate.instant('file_too_large') + ' (max. 20 Mo)');
```
Comment (gjn) [inline src/components/map/KmlService.js]: Dito.

File: src/components/map/MapService.js [hunk 3]
```diff
@@ -240,6 +244,10 @@ goog.require('ga_urlutils_service');
           updateDelay: {
             writable: true,
             value: null
+          },
+          userVisible: {
+            writable: true,
+            value: olLayer.getVisible()
```
Comment (gjn) [inline src/components/map/MapService.js]: Please help me: why this change?

Comment (oterral) [inline src/components/map/MapService.js]: It's happening only for layer added asynchronously (kml file).

For these layers we pass the visible property directly in the constructor before the call to `gaDefineLayerProperties` so the overrided `setVisible` function is never called so `userVisible` is always set to true even if the visibility permalink says it should be false.

File: src/components/feedback/FeedbackDirective.js (hunk 1 above)
Comment (oterral) [inline src/components/feedback/FeedbackDirective.js]: Changed to MB

File: src/components/map/MapService.js (hunk 3 above)
Comment (gjn) [inline src/components/map/MapService.js]: so it's for kml that should be invisible via PL?

**Comment by gjn** [comment]
If I'm correct, please merge once jenkins is happy.

File: src/components/map/MapService.js (hunk 3 above)
Comment (oterral) [inline src/components/map/MapService.js]: Not only KML I just saw that concerns also with External WMS.  That concerns all layers which set initial visibility with the layer's constructor and not with the overrided setVisible method.

Comment (gjn) [inline src/components/map/MapService.js]: Understood. Thanks alot for the explanation.

**Pull request merged**